packages/fxa-shared/l10n/rtl-locales.json

```diff
@@ -0,0 +1,18 @@
+[
+  "ar",
+  "ckb",
+  "dv",
+  "he",
+  "he-IL",
+  "ks",
+  "ps",
+  "fa",
+  "syr",
+  "ur",
+  "ug",
+  "pa",
+  "pa-IN",
+  "pa-PK",
+  "yi",
+  "arc"
+]
```

Comment:
@bcolsson Could you spot check that the codes above are indeed RTL languages? We don't support all of them, but aiming for decent completeness in case we add in support for more RTL languages at some point down the road.

Reply:
Thanks for the ping. @vpomerleau - I responded in a different comment/

Reply:
Thanks @bcolsson and @sardesam! I've updated the list with the changes you recommended